src/store.js

```diff
@@ -0,0 +1,12 @@
+import { applyMiddleware, createStore } from "redux"
+//import logger from "redux-logger"
+import thunk from "redux-thunk"
+import promise from "redux-promise-middleware"
+import allReducers from "./reducers"
+
+process.env.NODE_ENV === 'production'
```

Comment:
Here's one example of how you could use `logger()` in dev, but not in production:

```js

const middlewareList = [promise(), thunk]

if (process.env.NODE_ENV !== 'production') middlewareList.push(logger())

const middleware = applyMiddleware(middlewareList)

```